src/server/saved_objects/migrations/README.md

```diff
@@ -0,0 +1,115 @@
+# Migrations
+
+This is the system that manages the versioning of saved object indices. It manages changes to the index, including mappings, index templates, and changes to the shape of documents from one version of Kibana to another.
+
+## Outstanding questions
+
+- Should we migrate all docs in the index between minor versions?
```

Comment:
I'd much rather have one way to do a migration than two.  Also, if you index-in-place, then there is no rollback.

Reply:
Agreed.